ngraph/test/runtime/interpreter/reference/detection_output.hpp

```diff
@@ -0,0 +1,644 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <cstddef>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "ngraph/shape.hpp"
+
+namespace referenceTest
```

Comment:
Can I ask you to add all reference implementations in the namespace as in original nGraph? It should help in future copy past these implementations in the nGraph library.

```
namespace ngraph
{
    namespace runtime
    {
        namespace reference
        {
```

Reply:
changed namespace